addon/templates/components/frost-button.hbs

```diff
@@ -1,3 +1,4 @@
+{{! Template for the frost-button component }}
```

Comment:
I don't understand why we are adding a comment like this in every component templates.